weblab/static/js/visualizers/editMetadata/editMetadata.js

```diff
@@ -15,16 +16,16 @@ function metadataEditor(file, div)
     this.modelDiv = $('<div></div>', {id: 'editmeta_modelvars_div'}).text('loading model...');
     this.ontoDiv = $('<div></div>', {id: 'editmeta_ontoterms_div'});
     otherContent = '<div class="clearer">\n'
-        + '<p><label for="id_tag">Version:</label>\n'
+        + '<p><label for="id_tag">Tag:</label>\n'
         + '<input type="text" name="tag" id="id_tag" placeholder="Optional short label for this version"/></p>\n'
         + '<p><label for="id_commit_message">Description of this version:</label>\n'
         + '<a class="pointer" id="dateinserter"><small>use current date</small></a>\n'
         + '<span id="versionaction"></span><br/>\n'
         + '<textarea cols="70" rows="3" name="commit_message" id="id_commit_message"></textarea>\n'
         + '<span id="commitmsgaction"></span></p>\n'
-        // + '<p><input type="checkbox" name="reRunExperiments" id="reRunExperiments"/>\n'
-        // + '<label for="reRunExperiments">rerun experiments involving previous versions of this model</label>\n'
-        // + '<small>(this might take some time)</small></p>\n'
+        + '<p><input type="checkbox" name="reRunExperiments" id="reRunExperiments"/>\n'
+        + '<label for="reRunExperiments">Re-run experiments involving the previous version of this model</label>\n'
```

Comment:
Should this be the previous versions of the model? `record_experiments_to_run` iterates through the commit's parents (i.e. ancestors) - I'm not sure which it is intended to be but seems unclear to me.

Reply:
Ah, I assumed commit.parents gave just the immediate parents (there could be more than 1 in a merge) rather than all ancestors! It should be just immediate parents (and for Web Lab, that typically means just one previous version).

Reply:
Now fixed